src/main/app/common/allowances/allowanceHelper.ts

```diff
@@ -0,0 +1,133 @@
+import { Partner } from 'claims/models/response/statement-of-means/partner'
+import {
+  AllowanceRepository,
+  DependantAllowanceType,
+  DisabilityAllowanceType,
+  LivingAllowanceType,
+  PensionAllowanceType
+} from 'common/allowances/allowanceRepository'
+import { AgeGroupType, Child, Dependant } from 'claims/models/response/statement-of-means/dependant'
+import { DisabilityStatus } from 'claims/models/response/statement-of-means/disabilityStatus'
+import { Income, IncomeType } from 'claims/models/response/statement-of-means/income'
+
+export interface AllowanceHelper {
+  getMonthlyPensionerAllowance (income: Income[], partner: Partner)
+  getMonthlyDependantsAllowance (dependants: Dependant)
+  getMonthlyLivingAllowance (defendantAge: number, partner: Partner)
+  getCarerDisableDependantAmount (dependant: Dependant, isCarer: boolean)
+  getDisabilityAllowance (defendantDisability: DisabilityStatus, partner: Partner)
+}
+
+export class ResourceAllowanceHelper implements AllowanceHelper {
```

Comment:
Maybe rename this `AllowanceCalculations` and get rid of the interface. Then I think it is OK to have your `StatementOfMeansCalculations` tightly coupled with `AllowanceCalculations` now that we have `AllowanceRepository` outside of `StatementOfMeansCalculations`.